genienlp/predict.py

```diff
@@ -121,6 +121,12 @@ def run(args, device):
                                      device=device,
                                      tasks=args.tasks,
                                     )
+    
+    if args.pred_languages[0] is not None:
+        model.set_decoder_start_token_id(args.pred_languages[0].split('+')[0])
+    else:
+        # use English as default
+        model.set_decoder_start_token_id('en')
```

Comment:
Without checking if the model is mbart first? Doesn't it unintentionally affect other HuggingFace models?

Reply:
It checks within the decoder_start_token_id method if model is mbart.
For TransformerSeq2Seq models it's always a pass.